gslib/tests/test_rsync.py

```diff
@@ -2835,6 +2835,26 @@ def _Check():
 
     _Check()
 
+
+  def test_rsync_files_with_whitespace(self):
+    """Test to ensure filenames with whitespace can be rsynced"""
+    filename = 'foo bar baz.txt'
+    local_uris = []
+    bucket_uri = self.CreateBucket()
+    tmpdir = self.CreateTempDir()
+    contents = 'File from rsync test: test_rsync_files_with_whitespace'
+    local_file = self.CreateTempFile(tmpdir, contents, filename)
+
+    expected_list_results = frozenset(['/foo bar baz.txt'])
+
+    """Tests rsync works as expected."""
```

Comment:
Make this a regular pound-sign-style comment please :)

Reply:
Whoops! Fixed. :)